vite.config.ts

```diff
@@ -25,6 +30,15 @@ export default defineConfig(({ mode }: UserConfig): UserConfig => {
     II_VERSION: `${process.env.II_VERSION ?? ""}`,
   };
 
+  const testAppCanisterId = existsSync(
```

Comment:
I don't think we want that tough... Otherwise the vite dev server can only be started when the test app is also deployed. And I would assume that for most development activities it is not needed.

We could introduce some sort of e2e configuration for starting the dev server though. Should I look into that?